hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -45,6 +45,10 @@ def foreman_server?
     module_enabled?('foreman')
   end
 
+  def katello_enabled?
+    module_enabled?('katello')
+  end
```

Comment:
This makes me wonder if we should replace all uses of `module_enabled?('katello')` but we didn't replace other uses of `module_enabled?`

Reply:
I will follow up with a PR not to muddy this one with more changes.